### PR TITLE
Migrate to use keepalive-tool from keepalive-glib-tools

### PR DESCRIPTION
**Pull request by kjokinie**
Moving to keepalive-tool use. Also some cleanups. Should be tagged as 1.1.0 once merged.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,7 +32,9 @@ install: all
 dist: clean
 	rm -rf $(PACKAGE)
```
Comment (thp) [inline Makefile]: Maybe use `git archive --format=tar --prefix=$(PACKAGE)/ | bzip2 >$(PACKAGE).tar.bz2` for this?

File: scripts/btrfs-balance [hunk 2]
```diff
@@ -73,83 +69,68 @@ report_progress()
...
 
 check_balance_need
-RETVAL=$?
-if test $RETVAL -eq 1; then
-	keepalive_cleanup 1
-fi
+NEED_BALANCE=$?
 
-if test $NEED_BALANCE == 1; then
+if test "$NEED_BALANCE" == 1; then
```
Comment (thp) [inline scripts/btrfs-balance]: Now that `NEED_BALANCE` is not used anywhere else (as far as I can see), this:

```
check_balance_need
NEED_BALANCE=$?
if test "$NEED_BALANCE" == 1; then
```

Can be collapsed into:

```
if check_balance_need; then
```

**Comment by thp** [comment]
Two minor comments, other than that LGTM, thanks for the nice cleanup :+1:

**Comment by kjokinie** [comment]
Version 2:
- Make dist target use git archive
- Drop NEED_BALANCE variable

**Comment by thp** [comment]
LGTM.

**Pull request merged**